pyphare/pyphare/pharein/maxwellian_fluid_model.py

```diff
@@ -1,7 +1,31 @@
 
+import numpy as np
 from ..core import phare_utilities
 from . import globals
 
+
+class fn_wrapper:
+    def __init__(self, fn):
+        self.fn = fn
+    def __call__(self, *xyz):
+
+        if not isinstance(xyz[-1], list):
```

Comment:
assumes this is called from c++
vector = list (pybind) reminder/comment

Reply:
this code was moved to https://github.com/PHAREHUB/PHARE/pull/324